ti/instrumentation/java/bazel.go

```diff
@@ -81,6 +81,80 @@ func (b *bazelRunner) AutoDetectTests(ctx context.Context, workspace string, tes
 	return tests, nil
 }
 
+func parseBazelTestRule(r string) (ti.RunnableTest, error) {
+	// r = //module:package.class
+	if r == "" {
+		return ti.RunnableTest{}, fmt.Errorf("empty rule")
+	}
+	n := 2
+	if !strings.Contains(r, ":") || len(strings.Split(r, ":")) < n {
```

Comment:
Nit: can you do regex instead? Same below

Reply:
Comment resolved in other PR.